qiskit/start_here.ipynb

```diff
@@ -19,62 +19,62 @@
     "Welcome Qiskitters.\n",
     "\n",
     "\n",
-    "These tutorials aim to explain how to use Qiskit. We assume you have installed Qiskit if not please look at [qiskit.org](http://www.qiskit.org) or the install [documentation](https://qiskit.org/documentation/install.html). \n",
+    "These tutorials aim to explain how to use Qiskit. We assume you have installed Qiskit; if not, please look at [qiskit.org](http://www.qiskit.org) or the install [documentation](https://qiskit.org/documentation/install.html). \n",
     "\n",
     "\n",
-    "We've collected a core reference set of notebooks in this section outlining the features of Qiskit. We will be keeping them up to date with the latest Qiskit version, currently 0.9. The focus of this section will be how to use Qiskit and not so much on teaching you about quantum computing. For those interested in learning about quantum computing we recommend the awesome learning paths we and the community have put together.\n",
+    "We've collected a core reference set of notebooks in this section outlining the features of Qiskit. We will be keeping them up to date with the latest Qiskit version, currently 0.9. The focus of this section will be how to use Qiskit and not so much on teaching you about quantum computing. For those interested in learning about quantum computing we recommend the awesome [educational material](https://www-dev.quantum-computing.ibm.com/support) we and the community have put together.\n",
     "\n",
```

Comment:
can we not link to the dev. Test the link to the new ibm quantum experience. https://quantum-computing.ibm.com/support

Reply:
link works in main website.